circuit_maintenance_parser/providers.py

```diff
@@ -152,6 +153,13 @@ class Telstra(GenericProvider):
     _default_organizer = "gpen@team.telstra.com"
 
 
+class Verizon(GenericProvider):
+    """Verizon provider custom class."""
+
+    _parser_classes: Iterable[Type[Parser]] = [HtmlParserVerizon1]
+    _default_organizer = "NO-REPLY-sched-maint@EMEA.verizonbusiness.com"
```

Comment:
maybe not the most "generic" email, but until we don't have other, it should work

Reply:
agreed, but the only one we have right now.